lib/agent0/examples/interactive_econ.py

```diff
@@ -0,0 +1,453 @@
+# %%
+"""Run experiments of economic activity."""
```

Comment:
I think a quick pseudocode docstring at the top here would be really helpful. Similar to how we write the bot policies. Answer things like what question is the experiment answering and what is the "procedure" it follows.